pootle/apps/pootle_statistics/models.py

```diff
@@ -25,11 +25,12 @@
 from pootle_store.util import FUZZY, TRANSLATED, UNTRANSLATED
 
 
-EDIT_COEF = 5.0/7
-REVIEW_COEF = 2.0/7
-SUGG_COEF = 0.2
-ANALYZE_COEF = 0.1
-SIMILARITY_THRESHOLD = 0.5
+EDIT_COEF = settings.POOTLE_SCORING_CONF.get('EDIT_COEF', 5.0/7)
```

Comment:
Since a default is already defined in Pootle settings I don't think we need to provide it here. It will be a maintenance burden.